oneflow/core/job/graph_scope_vars.cpp

```diff
@@ -23,6 +23,16 @@ std::atomic<bool>* GetGraphVerboseStepLr() {
   static std::atomic<bool> graph_verbose_step_lr{false};
   return &graph_verbose_step_lr;
 }
+
+std::atomic<int32_t>* GetGraphDebugMaxPyStackDepthVar() {
+  static std::atomic<int32_t> graph_debug_max_py_stack_depth{2};
```

Comment:
`atomic` should be replaced with `thread_local` in this file, because these var is not safe to update by multi-thread.